lib/src/core/base/shared_preference/shared_pref_key.dart

```diff
@@ -1,2 +1,2 @@
-const String USER_UID = '';
-const bool IS_LOGGED_IN = false;
+const String USER_UID = 'USER_UID';
+const String IS_LOGGED_IN = 'IS_LOGGED_IN';
```

Comment:
`IS_LOGGED_IN` should be `bool` type.

Reply:
These are the pref keys, so the key of where the bool is stored.

Reply:
@dinurymomshad yes, but values in shared_pref_key.dart should be used only as something like dictionary/array with keys used in application, values we can get by SharedPref.getStringAsync(KEY_NAME_EXAMPLE)